GeometryDashAPI/Levels/GameObjects/Specific/SpeedBlock.cs

```diff
@@ -14,26 +14,59 @@ public enum SpeedBlockId
     }
 
     [GameBlock(200, 201, 202, 203, 1334)]
-    public class SpeedBlock : Block
+    public class SpeedBlock : Portal, IComparable<SpeedBlock>
```

Comment:
`IComparable` in the class means a complete match.
Instead of we can use `IComparer` for specific needs

Reply:
Okay, I'll just left only `OrderBy` and array, because that's enough